### PR TITLE
Add kubectl ko log

### LINKED ISSUE #2400: ko can collect all component logs for further analysis
### Feature request

New ko command like `kubectl ko log` to collect all component logs for further analysis.

### Use case

1. When E2E intermittent failes, we cannot read the log and make it difficult to debug.
2. When incident occurs in customers, collecting all logs needs lots of operation and takes long time.

### PR DESCRIPTION
**Pull request by changluyi**
- [ ] Make sure you have followed [Kube-OVN Code Style](https://github.com/kubeovn/kube-ovn/blob/master/CODE_STYLE.md).

#### What type of this PR
Examples of user facing changes:
- Features
- Bug fixes
- Docs
- Tests

#### Which issue(s) this PR fixes:
Fixes #2400

### CODE REVIEW

**Review by oilbeater — CHANGES_REQUESTED** [review]

File: dist/images/kubectl-ko [hunk 1]
```diff
@@ -26,8 +26,14 @@ showHelp(){
   echo "  env-check    check the environment configuration"
   echo "  tuning {install-fastpath|local-install-fastpath|remove-fastpath|install-stt|local-install-stt|remove-stt} {centos7|centos8}} [kernel-devel-version]    deploy kernel optimisation components to the system"
   echo "  reload    restart all kube-ovn components"
+  echo "  log {kube-ovn|ovn|ovs|linux} {all|sub_component...} [tail_lines]
```
Comment (oilbeater) [inline dist/images/kubectl-ko:29]: Need a short hand to collect logs for all components

**Review by oilbeater — CHANGES_REQUESTED** [review]

File: dist/images/kubectl-ko [hunk 2]
```diff
@@ -936,10 +942,144 @@ env-check(){
...
+      kubectl exec $pod -n kube-system -- top -b -n 1 | head -n 20 | tail -n 10 || :
+    fi
+  done
+}
+
+
+log(){
+  component="$1"
+  sub_component="$2"
+  tail_lines=${3:-100}
```
[the remark this answers is not in the document]
Comment (changluyi) [reply dist/images/kubectl-ko:1015]: fixed

File: dist/images/kubectl-ko (hunk 1 above)
Comment (changluyi) [reply dist/images/kubectl-ko]: fixed

**Review by oilbeater — APPROVED** [review]

File: .github/workflows/build-x86-image.yaml [hunk 3]
```diff
@@ -364,8 +364,8 @@ jobs:
         run: make kubectl-ko-log
         uses: actions/upload-artifact@v3
         with:
-          name: kubectl-ko-log
-          path: kubectl-ko-log.tar
+          name: k8s-conformance-e2e-ko-log
+          path: k8s-conformance-e2e-ko-log.tar
```
Comment (zhangzujian) [inline .github/workflows/build-x86-image.yaml:368]:
```suggestion
          path: k8s-conformance-e2e-${{ matrix.mode }}-${{ matrix.ip-family }}-ko-log.tar
```

**Pull request merged**